docs/excel/custom-functions-parameter-options.md

```diff
@@ -7,17 +7,16 @@ localization_priority: Normal
 
 # Custom functions parameter options
 
-Custom functions are configurable with many different options for parameters:
-- [Optional parameters](#custom-functions-optional-parameters)
-- [Range parameters](#range-parameters)
-- [Invocation context parameter](#invocation-parameter)
+Custom functions are configurable with many different options for parameters.
 
 [!include[Excel custom functions note](../includes/excel-custom-functions-note.md)]
 
-## Custom functions optional parameters
+## Optional parameters
```

Comment:
Please update the `ms.date` attribute value within the metadata at the top of the article.

Reply:
fixed.